lib/stripe.js

```diff
@@ -93,6 +102,14 @@ Stripe.prototype = {
     this._setApiField('protocol', protocol.toLowerCase());
   },
 
+  /**
+   * @deprecated in a future major version. Use the config object instead:
```

Comment:
Tiny nit: this reads a little funny to me... should it clarify that it will be _removed_ in a future release?